apps/datahub/src/app/record/record-metadata/record-metadata.component.html

```diff
@@ -1,24 +1,24 @@
-<div *ngIf="(facade.error$ | async) === null">
+<div *ngIf="(metadataViewFacade.error$ | async) === null">
```

Comment:
I struggled a bit as a new commer because it wasn't explicit enough for me.
I don't need to make my brain work to find what I'm manipulating, an extrem opposite would have been to rename it `mdvf `.

It's my personnal point of view, I' down to follow the naming convention if there is any ?

Reply:
I understand that feeling! But I would say that usually we try to not rename the existing things (especially if they are used multiple times), unless there is a specific reason (ie if there could be a confusion with something that you added). But maybe it's just my own feeling. WDYT @jahow @Angi-Kinas @tkohr ?

Reply:
IMO this can stay like this, it's more explicit now.